src/main/java/org/tc/models/User.java

```diff
@@ -35,12 +35,13 @@
     private Role role;
     @OneToMany
     @JoinColumn(name = "userid")
+    private List<Course> ownerCourses;
+    @OneToMany
+    @JoinColumn(name = "userid")
```

Comment:
userid is almost senseless here. Why don't you use such names as ownerId, subscriberId or participantId?